backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -182,9 +184,15 @@ public class CsvValidatorUtils {
           "y", "yes",
           "n", "no",
           "u", UNKNOWN_CODE);
-  private static final Set<String> TEST_RESULT_VALUES =
+  private static final Set<String> ACCEPTED_LITERAL_TEST_RESULT_VALUES =
       Set.of(POSITIVE_LITERAL, "negative", "not detected", DETECTED_LITERAL, "invalid result");
```

Comment:
nit: could we replace these loose strings with variables like we have for the positive and detected cases? Would rather not have the inconsistencies here